### PR TITLE
Allow Symfony 5

### PR DESCRIPTION
**Pull request by maxhelias**
- I remove support for PHP 7.1 as she is no longer maintained and she don't support `iterable` type for compatibility with Symfony 5.

- Add PHP 7.3 and 7.4 tests

- Try different versions of Symfony

TODO :

- [ ] Fix Travis 😄

### CODE REVIEW

**Comment by maxhelias** [comment]
@GuilhemN I think we can merge this PR even if the Sf5 test fails because of FOSRestBundle. It should automatically go green at the next release of FOSRest

File: Form/Extension/DocumentationExtension.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ public function getExtendedType()
         return self::getExtendedTypes()[0];
     }
 
-    public static function getExtendedTypes()
+    public static function getExtendedTypes(): iterable
     {
         return [FormType::class];
```
Comment (mantis) [inline Form/Extension/DocumentationExtension.php]: I've not tried this (was just looking at symfony 5 status), but should the return statement not be:

return array(FormType::class);

the addition of array() is in the upgrade doc's example?

Comment (maxhelias) [reply Form/Extension/DocumentationExtension.php]: I do not understand what you mean :/

Comment (mantis) [reply Form/Extension/DocumentationExtension.php]: I happened to read https://github.com/symfony/symfony/blame/master/UPGRADE-5.0.md#L153 about 30 seconds before reading your PR. The symfony guide 'after' example added array()

**Comment by GuilhemN** [comment]
Thank you @maxhelias ! :)

File: Form/Extension/DocumentationExtension.php (hunk 1 above)
Comment (ThibaultVlacich) [reply Form/Extension/DocumentationExtension.php]: @mantis `array()` and `[]` are the same thing. It's just two different ways of writing arrays in PHP. `[]` being more modern.

Comment (mantis) [reply Form/Extension/DocumentationExtension.php]: argh, sorry - I missed the []

**Pull request merged**